Tests/Marketplace/upload_packs.py

```diff
@@ -293,23 +349,35 @@ def main():
     packs_artifacts_path = option.artifacts_path
     extract_destination_path = option.extract_path
     storage_bucket_name = option.bucket_name
+    private_bucket_name = option.private_bucket_name
     service_account = option.service_account
     specific_packs = option.pack_names
     build_number = option.ci_build_number if option.ci_build_number else str(uuid.uuid4())
     override_pack = option.override_pack
     signature_key = option.key_string
 
+    # google cloud storage client initialized
+    storage_client = init_storage_client(service_account)
+    storage_bucket = storage_client.bucket(storage_bucket_name)
+    private_storage_bucket = storage_client.bucket(private_bucket_name)
+    index_folder_path, index_blob = download_and_extract_index(storage_bucket, extract_destination_path)
+    private_index_path, _ = download_and_extract_index(private_storage_bucket,
+                                                       os.path.join(extract_destination_path, 'private'))
+
     # detect new or modified packs
     modified_packs = get_modified_packs(specific_packs)
+    private_packs = get_private_packs(private_index_path)
```

Comment:
since there is no use for `private_packs` until line 374,
please move this line down, just before the `try` so no exceptions can happen in between and live the directory un cleaned